controller/src/main/java/io/pravega/controller/eventProcessor/impl/EventProcessorGroupImpl.java

```diff
@@ -107,7 +107,7 @@ void initialize() throws CheckpointStoreException {
             if (!e.getType().equals(CheckpointStoreException.Type.NodeExists)) {
                 throw e;
             } else {
-                log.warn("reader group {} exists", eventProcessorConfig.getConfig().getReaderGroupName());
+                log.trace("reader group {} exists", eventProcessorConfig.getConfig().getReaderGroupName());
```

Comment:
this can be debug, doesnt need to be trace

Reply:
ok